src/models/siteinfo.js

```diff
@@ -0,0 +1,51 @@
+import mongoose from 'mongoose';
+import slugs from 'slugs';
+
+const section = new Schema({
+  section_heading: {
+    type: String,
+    required: true,
+  },
+  section_content: [{
+    type: String,
+    required: true,
+  }],
+
+});
+const contributors = new Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  position: {
+    type: String,
+    required: true,
+  },
+  contact_info: [{
```

Comment:
Right now this is just email, let's name it so that it's clear what it stores. We should probably also make this one not required. 

Also, if it's set we should validate that it's a valid email. Read up on validation here http://mongoosejs.com/docs/validation.html